### ISSUE TITLE
StartAt option when scheduling an orchestration is not working

### ISSUE THREAD

**Issue by ursaciuc-adrian** [microsoft/durabletask-dotnet#131]
Hello there!

I set up an HTTP trigger to initiate an orchestration after a few days (for example), but it appears that the StartAt property specified in the StartOrchestrationOptions is not being considered, causing the orchestration to start immediately.

```
var options = new StartOrchestrationOptions
{
    InstanceId = id,
    StartAt = DateTimeOffset.UtcNow.AddDays(3)
};

await client.ScheduleSingletonOrchestrationAsync(nameof(Orchestration), input, options);
```

As a workaround, I could create a timer within the orchestrator, but I believe the options mentioned earlier should also function properly. 
Is there a mistake in my implementation that I am not recognizing?

I have also tried this in the provided sample, but the result is the same.

```
[Function(nameof(StartHelloCitiesUntyped))]
public static async Task<HttpResponseData> StartHelloCitiesUntyped(
    [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
    [DurableClient] DurableTaskClient client,
    FunctionContext executionContext)
{
    ILogger logger = executionContext.GetLogger(nameof(StartHelloCitiesUntyped));

    string instanceId = await client.ScheduleNewOrchestrationInstanceAsync(nameof(HelloCitiesUntyped), null, new StartOrchestrationOptions
    {
        StartAt = DateTimeOffset.UtcNow.AddDays(3)
    });
    logger.LogInformation("Created new orchestration with instance ID = {instanceId}", instanceId);

    return client.CreateCheckStatusResponse(req, instanceId);
}
```

**Comment by ursaciuc-adrian** [comment]
Thanks for the help @jviau ! Can you please let me know in which version will this be included?

**Comment by jviau** [comment]
@ursaciuc-adrian this has been included in the v1.0.2 release.

**Comment by ursaciuc-adrian** [comment]
Hello @jviau 
I am not sure in which repo I am supposed to post this issue anymore, but the issue appeared again https://github.com/Azure/azure-functions-durable-extension/issues/2751
